metadata.rb

```diff
@@ -867,6 +867,101 @@
 * squid3 is restarted to apply the new proxy config.
 '
 )
+recipe(
+  'mh-opsworks-recipes::configure-capture-agent-manager-gunicorn',
+  'sets up start script for gunicorn with capture-agent-manager app
+
+this is relevant for the utility node, where the capture-agent-manager `cadash`
+should run.
+
+=== attributes
```

Comment:
nice! but for consistency it'd be nice to match capitalization of all the other headers (nitpicky, yes).